OpenOversight/app/static/js/tag.js

```diff
@@ -0,0 +1,48 @@
+function isNumeric(value) {
+    return /^\d+$/.test(value)
+}
+
+$(document).ready(function () {
+    const img = $("#face-img")
+    const frame = $("#face-tag-frame")
+
+    const frameWidth = img.data("width")
+    const frameHeight = img.data("height")
+    const frameLeft = img.data("left")
+    const frameTop = img.data("top")
+
+    if ([frameWidth, frameHeight, frameLeft, frameTop].every(isNumeric)) {
+        img.one("imageLoaded", function () {
+            // To prevent hi-res images from taking over the entire page, the image is being shown
+            // in a responsive element. This means we need to compute the tag frame dimensions
+            // using percentages so it will scale if the page size changes.
+
+            // To do this, we're dividing the tag dimensions by the image's actual width/height and
+            // multiplying by 100.
+            const tagWidth = parseInt(frameWidth) / img[0].naturalWidth * 100
```

Comment:
Turn the `100` into a variable so we don't have the magic number sort of thing.

Reply:
I don't agree that 100 is a magic number in the context of percentages (e.g. https://codereview.stackexchange.com/questions/44047/100-is-a-magic-number). Perhaps we can rename the variable to make it clear its a percent value